### PR TITLE
squeezelite: create addon and dependencies

### PR DESCRIPTION
**Pull request by awiouy**
Hello,
This compiles and runs on my RPi2.
I nevertheless need help to copy the bins/libs to the appropriate locations.
Thank you.

### CODE REVIEW

**Comment by awiouy** [comment]
@CvH
Updated these: now they build for RPi2 after make clean, mpg123 is statically linked, and are included in multimedia tools.

**Comment by awiouy** [comment]
Hm... I forgot to update PKG_VERSION and PKG_RELEASE of multimedia_tools

**Comment by awiouy** [comment]
@CvH 
I had tested build squeezelite after make clean.
When I tested create_addon multimedia-tools after make clean, I noticed that it failed because libmediainfo depends on zlib.
I fixed that.
Ready for review and/or merge

File: packages/addons/addon-depends/multimedia-tools-depends/depends/libmediainfo/package.mk [hunk 1]
```diff
@@ -18,13 +18,13 @@
 
 PKG_NAME="libmediainfo"
 PKG_VERSION="0.7.83"
-PKG_REV="1"
+PKG_REV="2"
```
Comment (CvH) [inline packages/addons/addon-depends/multimedia-tools-depends/depends/libmediainfo/package.mk]: I don't think this is needed. @lrusak ?

Comment (lrusak) [inline packages/addons/addon-depends/multimedia-tools-depends/depends/libmediainfo/package.mk]: not needed but it's harmless so whatever :)

**Pull request merged**